src/devtools/packages/devtools-reps/object-inspector/utils/keyValue.tsx

```diff
@@ -2,15 +2,15 @@ import { ValueFront } from "protocol/thread";
 import { assert } from "protocol/utils";
 import { IItem, isValueLoaded, Item, LabelAndValue, ValueItem } from ".";
 import { ObjectInspectorItemProps } from "../components/ObjectInspectorItem";
-const PropRep = require("../../reps/prop-rep");
+import PropRep from "../../reps/prop-rep";
 
 export class KeyValueItem implements IItem {
   readonly type = "key-value";
-  name: string;
-  path: string;
-  key: ValueFront;
-  value: ValueFront;
-  childrenLoaded: boolean;
+  readonly name: string;
+  readonly path: string;
+  readonly key: ValueFront;
+  readonly value: ValueFront;
+  readonly childrenLoaded: boolean;
```

Comment:
Ah, so we _were_ mutating before and that was causing React not to re-render?

Reply:
No, I just added this to clarify that these objects should never be mutated.